snyk/models.py

```diff
@@ -430,6 +430,7 @@ class Project(DataClassJSONMixin):
     issueCountsBySeverity: IssueCounts
     imageTag: Optional[str] = None
     imageId: Optional[str] = None
+    tags: Optional[List[Any]] = field(default_factory=list)
```

Comment:
May I propose we make `tags` a type produced by `Manager.factory`?
if this type would have `all(), add(), delete()` methods then the calls on project would look like:
`project.tags.all()` - list tags for a project
`project.tags.add(**kwargs)` - add tags for project, unfortunately this will require N calls as the tags POST API allows specifying a single key and value attributes
`project.tags.delete(**kwargs)` - delete tags for project, unfortunately this will require N calls to the tags DELETE API for the same reasons above.